scripts/agents/includes/check-template-labels.js

```
@@ -86,19 +189,143 @@ function getTemplateLabels() {
+  }
+
+  if (Array.isArray(labelsValue)) {
+    return labelsValue.flatMap((label) =>
+      typeof label === "string" ? [label.trim()] : [],
+    );
+  }
+
+  if (typeof labelsValue === "string") {
+    return labelsValue
+      .split(",")
+      .map((label) => label.replace(/['"\[\]]/gu, "").trim())
```

ESLint

[error] Unnecessary escape character: \[.

(no-useless-escape)